addons/talo/apis/channels_api.gd

```diff
@@ -138,3 +138,15 @@ func send_message(channel_id: int, message: String) -> void:
 		},
 		message = message
 	})
+
+
+# Structs
+class ChannelPage:
+	var channels: Array[TaloChannel]
+	var count: int
+	var last_page: bool
+
+	func _init(channels: Array[TaloChannel], count: int, last_page: bool) -> void:
+		self.channels = p_channels
+		self.count = p_count
```

Comment:
you'll need to update these variable names to remove the `p_` prefix (to match the params)

Reply:
Already fix now.